docs/application/tizen-studio/release-notes/4-6-release-notes.md

```diff
@@ -0,0 +1,125 @@
+    - On Ubuntu, if the graphics driver is out-of-date, your Ubuntu desktop session occasionally logs out while launching Emulator Manager, or the emulator skin is displayed improperly. Verify the prerequisites and upgrade to the latest graphics driver.
+  - On Windows, depending on your OS theme (such as Non-Aero themes and Windows XP themes), a display surface can be erased for a while if the emulator window is covered with another window. If you click the emulator window, the display surface runs correctly again.
+  - On Windows, if an error with message "failed to allocate memory" occurs while executing the emulator, try the following:
+    - Close some other programs and try to launch the emulator again.
+    - If the RAM size is set to 768 or 1024 MB for the VM in Emulator Manager, change it to 512 MB.
+    - Increase the user area of the virtual memory in the system to 3 GB by entering the **bcdedit /setincreaseuserva 3072** command on the console with administrator rights (only in Windows 7), and reboot.
+  - If you use a MacBook Pro that has both Intel HD and NVIDIA&reg; GPUs, the emulator can terminate unexpectedly when you execute the emulator with OpenGL ES version 1.1 or 2.0. Verify the emulator configuration in Emulator Manager and on the **General** tab in the emulator configuration window, set OpenGL ES version to version 2.0, or to version 3.0.
+  - When you launch Emulator Manager in the Tizen IDE, the shortcut image of Emulator Manager may not be displayed properly.
+  - Basic Web applications are not installed on SD cards.
+  - To use Tizen Emulator in Tizen platform 3.0 or lower, disable the CPU VT option in the **HW Support** tab of Emulator Configuration.
+- CLI and SDB
+  - Tizen Studio does not support the [Smart Development Bridge](../common-tools/smart-development-bridge.md) (SDB) bash auto-completion on Windows (it is available on Ubuntu and macOS).
+- Dynamic Analyzer
+  - When analyzing applications on commercial devices running Tizen 3.0, newly-released or after a firmware update, the following problems exist:
+    - The Core Frequency information is not shown.
+    - The screenshots on the scene transitions feature will not work.
+  - When analyzing applications on the Tizen 4.0 emulator or reference device, the startup profiling information is not shown.
+  - The UI Hierarchy viewer feature and startup profiling are not performed simultaneously.
+  - The Dynamic Analyzer cannot show lifecycle information for Web applications.
+  - Widget applications cannot be profiled with the Dynamic Analyzer. They are hidden in the application list on the toolbar for all Tizen platforms, except Tizen 2.3.2.
+  - The Dynamic Analyzer sometimes gets stuck which is caused by an internal security checking program.
+- Visual Studio
+  - NUI XAML application build error can be resolved by doing clean and then build again.
```

Comment:
Line 123: Should the word “clean” and “build” stay as it is considering it specifies the feature itself? Or should we restructure the sentence?